AdminUi/test/AdminUi.Tests.Integration/Features/IdentityOverviews/GET.feature

```diff
@@ -1,9 +1,9 @@
 ﻿@Integration
-Feature: GET IdentityOverviews
+Feature: GET Identities
 
-User requests an IdentityOverview List
+User requests an Identity Overview List
 
 Scenario: Requesting an Identity List
-	When a GET request is sent to the /Identities/Overview endpoint
+	When a GET request is sent to the /Identities endpoint
 	Then the response status code is 200 (OK)
-	And the response contains a paginated list of Identity Overviews
+	And the response contains a non empty list
```

Comment:
Honestly? My comment was aiming for you adding some assertions in order to match the Step description, and not the other way round.